src/agent0/chainsync/analysis/calc_position_value.py

```diff
@@ -98,7 +125,17 @@ def calc_single_closeout(
                 fp_out_value = interface.calc_close_long(amount, maturity, hyperdrive_state)
         # Rust Panic Exceptions are base exceptions, not Exceptions
         except BaseException as exception:  # pylint: disable=broad-except
-            logging.info("Chainsync: Exception caught in calculating close long, ignoring: %s", exception)
+            logging.info(
+                "Chainsync: Exception caught in calculating close long: %s\nApproximating with spot price.", exception
+            )
+
+            # TODO: We can use the rust `calculate_market_value_*` functions once
+            # https://github.com/delvtech/hyperdrive-rs/pull/153 is merged.
+            # Long value = users_longs * spot_price * term_remaining
+            normalized_time_remaining = calc_scaled_normalized_time_remaining(
+                maturity, FixedPoint(checkpoint_time), FixedPoint(position_duration)
```

Comment:
No need to cast to `FixedPoint` here, the interface should already have this as FixedPoint. Removing it will also ensure type safety, just in case `checkpoint_time` and `position_duration` is an int.

Reply:
the interface provides ints 
<img width="1192" alt="image" src="https://github.com/user-attachments/assets/7fdd4563-4be9-4a86-8ce5-6ee59b81d7c6">

Reply:
Scratch that, both `checkpoint_time` and `position_duration` should be integers, representing epoch time instead of a `FixedPoint` number.